### PR TITLE
Add 1.0 Docker images, and upgrade all the images to follow the Quick Starts for each language.

### PR DESCRIPTION
**Pull request by markmandel**
I updated the language version for each one as well to what is latest.

I was tempted to remove the onbuild images - simply because they expanded the support required, but I updated the version numbers they depended on and left them in. Let me know your thoughts.

Please let me know if there is any adjustment / refinement you would like to see.

### CODE REVIEW

File: 1.0/python/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# gRPC Python Docker image
+
+This is the official docker image for the Python facility of grpc.  For an
```
Comment (nathanielmanistaatgoogle) [inline 1.0/python/README.md]: gRPC when written in prose should always be styled as "gRPC".

File: 1.0/python/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+overview and usage examples, see the grpc python documentation.
```
Comment (nathanielmanistaatgoogle) [inline 1.0/python/README.md]: Styling here should be "gRPC Python".

File: 1.0/python/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+FROM python:2.7
+
+ENV GRPC_PYTHON_VERSION 1.0.0
+RUN python -m pip install --upgrade pip
+RUN pip install grpcio==${GRPC_PYTHON_VERSION} grpcio-tools==${GRPC_PYTHON_VERSION}
```
Comment (nathanielmanistaatgoogle) [inline 1.0/python/Dockerfile]: Missing newline at end-of-file (here and in at least one other file in this pull request).

**Review by nathanielmanistaatgoogle — CHANGES_REQUESTED** [review]

**Comment by markmandel** [comment]
Updated the Dockerfiles so that they all have linebreaks, and fixed up all the readme's in the `1.0` folder such that they state "gRPC" (worth nothing that in `0.10` and `0.11` folders, they are still "grpc")

Please let me know if you have any further feedback.

File: 1.0/python/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# gRPC Python Docker image
+
+This is the official docker image for the Python facility of gRPC.  For an
+overview and usage examples, see the gRPC python documentation.
```
Comment (nathanielmanistaatgoogle) [inline 1.0/python/README.md]: "gRPC Python" on this line too.

**Comment by nathanielmanistaatgoogle** [comment]
I was just jumping on some cosmetic stuff; @nicolasnoble will be the reviewer of substance for this change (or he will route this change to someone else who will be).

**Comment by markmandel** [comment]
No worries. Good catches. Fixed up now. :+1:

File: 1.0/cxx/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+# install grpc
+RUN cd /var/local/git/grpc && \
+    git submodule update --init && \
+    make && \
+    make install && make clean
+
+#install protoc
+RUN cd /var/local/git/grpc/third_party/protobuf && \
+    make && make install && make clean
```
Comment (nicolasnoble) [inline 1.0/cxx/Dockerfile]: Missing end of line.

**Review by nicolasnoble — CHANGES_REQUESTED** [review]
One tiny nit, otherwise lgtm.

**Comment by markmandel** [comment]
Added newline. Let me know if you would like a rebase to clean up the commit history or not :smiley:

**Comment by nathanielmanistaatgoogle** [comment]
I favor this squashing; @nicolasnoble will be the one to actually decide.

**Comment by markmandel** [comment]
@nicolasnoble What do you reckon? Shall I squash it, and then it can be merged? :smile:

**Review by nicolasnoble — APPROVED** [review]

**Comment by nicolasnoble** [comment]
I don't really care either way - that repository has less importance when it comes to history, so squash, don't squash, do what you prefer :)

**Comment by markmandel** [comment]
@nathanielmanistaatgoogle Do you feel strongly either way? I don't mind. I just want to get this through the door? :smile:

**Comment by nathanielmanistaatgoogle** [comment]
Squash.

**Comment by nathanielmanistaatgoogle** [comment]
Also for extra bonus points adhere to [these guidelines](http://chris.beams.io/posts/git-commit/#seven-rules).

**Comment by markmandel** [comment]
@nathanielmanistaatgoogle Squashed and updated the commit message.

(Love that guideline. Stealing for future use)

**Review by nathanielmanistaatgoogle — APPROVED** [review]

**Comment by markmandel** [comment]
Thank you!

**Pull request merged**